X-Guardian/terraform-docs
# Remove extra newlines between comments and inputs/outputs to fix Mark…

…DownLint warnings (#66)

* Remove extra blank lines between Comments and Inputs/Outputs

* Update markdown testdata to remove extra blank lines

* Move HasInputs check

* Remove spurious blank line from main.tf example

* Test Commit

* Test commit reversal

* Remove spurious blank line from main.tf

* Fix doc_test.go to remove expected spurious blank lines

* Remove spurious blank lines from json and pretty tests

### Prerequisites

Put an `x` into the box(es) that apply:

- [ ] This pull request fixes a bug.
- [ ] This pull request adds a feature.
- [ ] This pull request introduces breaking change.

For more information, see the [Contributing Guide](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

### Description

Describe what this pull request achieves. Ensure you have read the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md) document before submitting.

### Issues Resolved

List any existing issues this pull request resolves.

### Checklist

Put an `x` into all boxes that apply:

- [ ] I have read the [Contributing Guidelines](https://github.com/segmentio/terraform-docs/tree/master/CONTRIBUTING.md).

#### Tests

- [ ] All tests pass when I run `make test`.

#### Documentation


#### Code Style
